README.md

```diff
@@ -37,6 +37,30 @@
 - It's also recommended to install [Docker Support for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=PeterJausovec.vscode-docker) to manage your IoT Edge Docker images, containers and registries. 
 - The current user must have permissions to manage Docker. By default this is not the case on Linux, check the [FAQ section](#FAQ) to avoid `permission denied` errors.
 
+  #### C# module prerequisites
```

Comment:
How about put language requirement before the extension requirement?

Reply:
OK